h/views/accounts.py

```diff
@@ -284,10 +294,9 @@ def _reset_password(self, user, password):
         self.request.session.flash(
             jinja2.Markup(
                 _(
-                    "Your password has been reset. "
-                    'You can now <a href="{url}">login</a> with your new '
-                    "password."
-                ).format(url=self.request.route_url("login"))
+                    "Your password has been reset. You can now login with "
+                    "your new password."
+                )
```

Comment:
This flash message is now shown on the login page so it doesn't need to link to the login page